CMakeLists.txt

```diff
@@ -4,8 +4,24 @@ project(tinyspline)
 # Turn on folders
 set_property(GLOBAL PROPERTY USE_FOLDERS ON)
 
+# Detect tinyspline being built as main project
+if(CMAKE_SOURCE_DIR STREQUAL "${tinysplice_SOURCE_DIR}")
```

Comment:
tinysplice?

Reply:
Is it ok if I push a commit with the fixes or do you prefer a force push with the fixed commits?

Reply:
Whatever is the easiest for you.